.taskcluster.yml

```diff
@@ -118,6 +118,7 @@ tasks:
           && git checkout {{ event.version }}
           && python tools/taskcluster/get-adjust-token.py
           && python tools/l10n/filter-release-translations.py
+          && python tools/l10n/check_locales.py
```

Comment:
Will it stop if output is 1? (`exit(1)`)
Is it correct place to add? :)

Reply:
I'm not sure – can you test this? Your PRs should build on task cluster so you should be able to add 1's, push a PR, and see if it stops execution correctly.

That being said, stopping execution on 1 makes sense to me.